app/views/pages/chose_no_to_voluntary_registration.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@()(implicit request: Request[_], messages: Messages)
+@import uk.gov.hmrc.play.views.html.{helpers => govHelpers}
+
+@main_template(title = messages("pages.voluntary.registration.no.title")) {
+
+
+    <h1 class="form-title heading-xlarge" id="pageHeading">@messages("pages.voluntary.registration.no.heading")</h1>
+
+    @govHelpers.form(action = controllers.routes.VoluntaryRegistrationController.showChoseNoToVoluntary) {
```

Comment:
this doesnt need to have this gov helper wrapper. it isnt a form. remove this wrapper and remove unneccesary lines in file - too much white space